webapp/graphql/Registration.js

```diff
@@ -7,6 +7,7 @@ export const SignupVerificationMutation = gql`
     $password: String!
     $about: String
     $termsAndConditionsAgreedVersion: String!
+    $language: String
```

Comment:
This is going in the right direction. Could you also add `locale` as allowed input parameter to `UpdateUser`, please?

Reply:
mit dem nächsten PR würde ich das gern + anwendungsfall erledingen um die sache nicht wieder so in die länge zu ziehen

Reply:
@ogerly OK, could you rename `language` to `locale`? I'm OK to merge this then.

@mattwr18 what is the better name here? :point_up: I researched `locale` a little and it seems it denominates the language code + country. https://en.wikipedia.org/wiki/Language_code
So it's actually `en-US` and not only `en`. https://en.wikipedia.org/wiki/Locale_(computer_software) We only use the language code. `locale` still good?

Reply:
@roschaefer **language** to **_locale_** change